docs/openvpn.md

```diff
@@ -0,0 +1,164 @@
+# OpenVPN
```

Comment:
Could you please add link(s) to resources containing extended info for commands and configs?
For example: the dockerhub page for the image used, a setup guide, description of unobvious config changes.
Don't need to be a link for every paragraph, just for things you think are important.